migrations/20210201160014-fix-opencollective-paymentMethod-currency.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    // Fix Payment Methods currency not matching the Host currency
```

Comment:
We're migrating but the currency should have no impact on the transaction in the future.